src/components/ha-service-control.ts

```diff
@@ -174,7 +176,29 @@ export class HaServiceControl extends LitElement {
         .value=${this.value?.service}
         @value-changed=${this._serviceChanged}
       ></ha-service-picker>
-      <p>${serviceData?.description}</p>
+      <div class="description">
+        <p>${serviceData?.description}</p>
+        ${this.value?.service
+          ? html` <a
+              href="${documentationUrl(
+                this.hass,
+                "/integrations/" + computeDomain(this.value?.service)
+              )}"
+              title="${this.hass.localize(
+                "ui.panel.developer-tools.tabs.services.help"
```

Comment:
This component is used in more panel then just the `developer-tools` panel, the translation should be in the component section.